ingesters/auditlog/auditlogingester_test.go

```diff
@@ -0,0 +1,74 @@
+package auditlog_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+
+	"github.com/metal-toolbox/audito-maldito/ingesters/auditlog"
+	"github.com/metal-toolbox/audito-maldito/internal/health"
+)
+
+func TestIngest(t *testing.T) {
+	t.Parallel()
+	tmpDir, err := os.MkdirTemp("", "")
+	if err != nil {
+		t.Errorf("failed to initialize tests: Could not create %s dir", tmpDir)
+	}
```

Comment:
we've been using testify [1] in other parts of the repo, I suggest you migrate these checks to that. It would look as follows:

```go
assert.NoError(t, err, "failed to initialize tests")
```

[1] https://pkg.go.dev/github.com/stretchr/testify/assert

Reply:
will do